powerauth-restful-security-javaee/src/main/java/io/getlime/security/powerauth/rest/api/jaxrs/service/v3/UpgradeService.java

```diff
@@ -92,7 +92,7 @@ public EciesEncryptedResponse upgradeStart(EciesEncryptedRequest request, PowerA
             response.setEncryptedData(upgradeResponse.getEncryptedData());
             return response;
         } catch (Exception ex) {
-            logger.warn("PowerAuth upgrade start failed", ex);
+            logger.warn("PowerAuth upgrade start failed, error: {}", ex.getMessage());
```

Comment:
Let's pass `ex` as a parameter to the new exception to keep the cause somewhere.